javascript/Makefile

```diff
@@ -0,0 +1,3 @@
+buildjs:
```

Comment:
this is the first step to automatically create the and copy the bundle.js at the right place.
I also thought about creating the bundle.js as a softlink but this might be an issue on different OS and/or when building wheels (idk whether it really causes problems there, I just remember that linked files cannot be copied easily into docker containers and think we might run into similar issues here). 

Maybe in the future we'll have one workflow that copies the js file over and then runs the tests/plots directory to make sure that even with the new bundle everything is as expected. Or we'll create a pre-commit flow that compares the two bundle.js and enforces that they're equal. Just some thoughts, curious for your input here